apps/ecommerce/frontend/package.json

```diff
@@ -28,8 +28,8 @@
     "lint": "eslint --max-warnings=0 .",
     "eject": "react-scripts eject",
     "create-app-definition": "contentful-app-scripts create-app-definition",
-    "deploy": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEFINITIONS_ORG_ID} --definition-id 6wFPGvNI463EN5xF7AwIwA --token ${CONTENTFUL_CMA_TOKEN}",
-    "deploy:test": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${TEST_ORG_ID} --definition-id ${TEST_APP_ID} --token ${CONTENTFUL_CMA_TOKEN}"
+    "deploy": "aws s3 cp ./build/ s3://cf-apps-ecommerce/$(git rev-parse --short HEAD) --recursive",
```

Comment:
we no longer use Contentful's app bundles to host the ecommerce app frontend – this is now in it's own S3 bucket which drastically simplified the work involved around making https://ecommerce-app.ctfapps.net/ vanity URL work for the other "wrapper" apps. (if we use app bundles, there is a lot of unnecessary complexity around signatures, etc)